articles/basics/introduction-to-fuse.md

```diff
@@ -165,7 +165,7 @@ Just like how an apple and a truck have different properties in real life, diffe
 
 One last note on properties: when we have an _object_ with *opening* & *closing* tags we only add the properties to the *opening* tag, so:
 
-```
+```xml
       / Only need properties on this one.
      v
  <StackPanel Orientation="Horizontal">
```

Comment:
I on't think we want to mark these as xml; I intentionally skipped over those who have drawing in them, that will just confuse the syntax highlighter... I think you should do the same...

Reply:
Does it not look normal, except with the "drawing"?

Reply:
Check the one with and without: https://github.com/fuse-open/docs/blob/cbd9542256022de05fc4d76c3b276a3dc29a609c/articles/basics/introduction-to-fuse.md

Reply:
What's actually needed for those "drawings" are images lol

Reply:
I shall draw some up for that page...

Reply:
Yeah, good point. I guess some custom HTML with some annotations would be even better?

Reply:
What kinda annotations are you thinking of? Its pretty quick to draw up image versions.

Reply:
The problem with images is that they hard-code fonts and colors, making it harder to tweak the look without things looking inconsistent.

I was thinking of injecting stuff like [Bootstrap Popovers](https://getbootstrap.com/docs/4.3/components/popovers/) into the indevidual span-elements generated by the syntax-highlighter. It would probably require a bit of custom JavaScript, or rendering out the generated HTML. But either of those will avoid the hardcoding issue...

Reply:
I get it but its only 3 images and to redo them isn't hard plus I don't imagine us rebranding anytime soon.

Reply:
Images are done, you can still do the popover injection thing if you want to, I won't mind.

Reply:
Fonts (and how they render) can vary from platform to platform or even browser to browser, and resolution to resolution. Especially high-dpi screens (which is something I use pretty much exclusively) gives me a bit of pause with this solution.

I also don't like arguments on the form "this isn't so bad, we only do it X times", because the sum of many small things quickly becomes large.

Finally, I don't think that us not rebranding anytime soon is a good argument; things like this gets *harder* to remember with time, not easier. So it's likely to slip a year down the line or something.

I'm probably making a bit of a "perfect is the enemy of good"-mistake here, but let's at least let's let things sink in a bit first...  In the mean time could you just leave these tags as-is in this PR, and rather send a separate one for the image-stuff? That way they don't hold the other fixes "hostage"? :smile:

Reply:
Ok, sorted, reverted the change and saved the images in a separate branch 😉 👍